scripts/start-setupForgeApiMods

```diff
@@ -97,6 +97,7 @@ modFileByProjectID(){
     # Checking for a individual release type input, if not use global
     if [ $project_id_release_type ]; then
       updateReleaseNumber $project_id_release_type
+      unset $project_id_release_type 
```

Comment:
Bug that was fixed.

Reply:
It looks like `unset` takes a variable name, so would be
```suggestion
      unset project_id_release_type 
```